src/sgame/sg_main.cpp

```diff
@@ -261,6 +261,7 @@ Cvar::Cvar<float> g_bot_fov("g_bot_fov", "bots' \"field of view\"", Cvar::NONE,
 Cvar::Cvar<int> g_bot_chasetime("g_bot_chasetime", "bots stop chasing after x ms out of sight", Cvar::NONE, 5000);
 Cvar::Cvar<int> g_bot_reactiontime("g_bot_reactiontime", "bots' reaction time to enemies (milliseconds)", Cvar::NONE, 500);
 Cvar::Cvar<bool> g_bot_infinite_funds("g_bot_infinite_funds", "give bots unlimited funds", Cvar::NONE, false);
+Cvar::Cvar<bool> g_bot_ignore_momentum("g_bot_ignore_momentum", "allow bots to ignore momentum, but not other restrictions", Cvar::NONE, false);
```

Comment:
I would prefer the name `g_bot_infiniteMomentum` since it does something similar to infinite funds. (But there shouldn't be an underscore after `infinite` since we may want to officially make `_` a namespace separator in tab completion.)

The `but not other restrictions` part of the description seems unnecessary.

Reply:
> Ok, but, currently, there seems to be an overall convention to use snake_case in cvar names? 

No, there's only a couple cvars like that. There's a mix of g_bot_foobar, g_bot_fooBar, and g_bot_foo_bar.


> Also, it would play well with the idea of namespaces to have both `g_bot_infinite [ funds | momentum ]` no?

2 cvars is far too few to deserve their own namespace.

Reply:
ok then. I'll rename the other one in the same time then.

Reply:
I'm too lazy to update other stuff. I'll just use the g_bot_infiniteMomentum as suggested.

Reply:
Note that we really need to write down those "conventions" or rather, define conventions, document them, and migrate codebase to those. With that addition of g_bot_infiniteMomentum I just added "yet another convention" in the code (other stuff in g_bot do *not* use uppercase I think).
I did that so that the code will be merged, and won't do another update on this PR for nitpicking, because this 1) is very annoying and 2) makes it more likely to have broken UI (because typos in a subrepo, with no means to check sanity)

If there are actual bugs or real improvements to do, though, I'll be ok with those.

Reply:
Is it good enough now?

Reply:
Yes